modules/openapi-generator/src/main/resources/go-server/README.mustache

```diff
@@ -28,3 +28,14 @@ To run the server, follow these simple steps:
 go run main.go
 ```
 
+To run the server in a docker container
+```
+docker build --network=host -t petstore .
+```
+
+Once image is built use
+```
+docker run --rm -it petstore 
```

Comment:
@etherealjoy should `petstore` be replaced by {{{packageName}}} instead?

Reply:
indeed, I have updated the commit.